hoot-js/src/main/cpp/hoot/js/conflate/matching/ScriptMatchCreator.cpp

```diff
@@ -727,6 +833,12 @@ CreatorDescription ScriptMatchCreator::_getScriptDescription(QString path) const
     Handle<Value> value = ToLocal(&plugin)->Get(featureTypeStr);
     result.baseFeatureType = CreatorDescription::stringToBaseFeatureType(toCpp<QString>(value));
   }
+  Handle<String> geometryTypeStr = String::NewFromUtf8(current, "geometryType");
```

Comment:
Added this so that every generic script now optionally identifies what geometry types it conflates. If not specified (see `PointPolygon.js`, we assume it conflates all geometry types and no optimization is done.